### PR TITLE
[IMP] legal: german translation of enterprise agreement

### PR DESCRIPTION
**Pull request by frfa1993**
I added an updated German translation for the German enterprise agreement and I edited the main legal .rst to add the corresponding PDF file.

### CODE REVIEW

File: content/legal/terms/i18n/enterprise_de.rst [hunk 1]
```diff
@@ -21,345 +21,452 @@ Odoo Enterprise Subscription Agreement (DE)
...
-.. _charges_de:
+Die Einzelheiten der Cloud-Hosting-Dienste sind in der Dienstgütevereinbarung unter
+https://www.odoo.com/cloud-sla beschrieben.
 
-5. Abgaben und Gebühren
+4.5 Support-Dienste
+-------------------
+Umfang
+++++++
+Während der Laufzeit dieser Vereinbarung kann der Kunde eine unbegrenzte Anzahl von kostenlosen
```
Comment (jcs-odoo) [inline content/legal/terms/i18n/enterprise_de.rst:225]:
```suggestion
4.5 Support-Dienste
-------------------

Umfang
++++++

Während der Laufzeit dieser Vereinbarung kann der Kunde eine unbegrenzte Anzahl von kostenlosen
```

File: content/legal/terms/i18n/enterprise_de.rst [hunk 2]
```diff
@@ -21,345 +21,452 @@ Odoo Enterprise Subscription Agreement (DE)
...
+    der Kunde manuell eingreifen muss.
 
 .. _upgrade_de:
 
-4.3 Upgrade-Dienstleistungen
-----------------------------
-
+4.3 Upgrade-Dienste
+-------------------
 .. _upgrade_odoo_de:
```
Comment (jcs-odoo) [inline content/legal/terms/i18n/enterprise_de.rst:172]:
```suggestion
4.3 Upgrade-Dienste
-------------------

.. _upgrade_odoo_de:

```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello @frfa1993 
Thanks for this PR!

As discussed previously with @AntoineVDV , please add white lines between each command. This includes headings and paragraphs.

Also, we'll have to rebase your branch onto 14.0 and create a new PR to change the target of the PR.

Thanks

**Comment by frfa1993** [comment]
Backported in #1453

**Pull request closed without merging**